modules/ROOT/pages/changelog.adoc

```diff
@@ -4,6 +4,78 @@
 
 NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].
 
+== 6.5 - 2023-06-01
+
+=== Added
+* New `table_merge_content_on_paste` option which disables the merging behaviour when pasting a table inside an existing table.
+* New optional `defaultExpandedIds` and `onToggleExpand` options to the `tree` component config.
+* New optional `defaultSelectedId` option to the `tree` component config.
+* New `help_accessibility` option which displays the keyboard shortcut to access the help functionality in the statusbar.
+* New `accordion` plugin with the `InsertAccordion` command.
+* New `accordion` and `accordion-toggle` icons.
+* New `details_initial_state` and `details_serialized_state` options.
+* New `init_content_sync` option that initializes the editor iframe using `document.write` instead of `srcdoc`.
+* New `newdocument_content` option that changes the content that will be set when using the `New document` button.
+* New `editable_root` option that can be set to `false` to prevent editing of the editors root element.
+* New `editor.setEditableRoot` API that sets the editable state of the editor root element.
+* New `editor.hasEditableRoot` API that returns `true` or `false` depending on the editable state of the editor root element.
+* New `EditableRootStateChange` event that gets dispatched when the state of the editable root is changed.
+
+=== Improved
+* Screen readers are now able to announce the highlighted menu item of link comboboxes.
+* Now `icon` field for dialog footer `togglebutton`s is not mandatory.
+* Toolbar buttons and menu items were not disabled when they couldn't be used on noneditable content.
+* Updated toolbar "More" button tooltip text from "More..." to "Reveal or hide additional toolbar items".
+* Help text displayed at **Help -> Help -> Keyboard Navigation** re-written.
+* These characters — *$*, *~*, *+*, *|*, *№*, and *`* — are now considered as punctuation marks. Therefore, they will not increase the word count.
+* Updated the `codesample` plugin dialog and `template` plugin dialog to use the 'listbox' component to match other dialogs.
+* Quickbar toolbars are now positioned in the middle of the selection horizontally, if the selection contains more than one table cell.
+* Exposed `dataTransfer` property of drag and drop events for elements with a `contenteditable="false"` attribute.
+
+=== Changed
+* The `caption`, `address` and `dt` elements were allowed to have non-inline children elements when the editor schema was set to `html4`.
+* SVG icons for back and foreground colors now uses class instead of id to identify SVG elements that should change color.
+* The `a` elements were allowed to have non-inline child elements when the editor schema was set to `html4`.
+
+=== Fixed
+* Right clicking on a merge tag would result in different highlighting across browsers.
```

Comment:
```suggestion
* Right clicking on a merge tag instance presented different highlighting depending on the host browser.
```

Reply:
Updated to reflect the incoming changes from the approved PR

Reply:
To quote my commit message:
> Incorporated copy-edit provided by @kemister85 in the currently open PR.